src/main/java/ui/driverConfig/DriverProperties.java

```diff
@@ -0,0 +1,37 @@
+package ui.driverConfig;
+
+import org.aeonbits.owner.ConfigFactory;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.edge.EdgeDriver;
+
+import static org.openqa.selenium.chrome.ChromeDriverService.CHROME_DRIVER_EXE_PROPERTY;
+import static org.openqa.selenium.edge.EdgeDriverService.EDGE_DRIVER_EXE_PROPERTY;
+
+public class DriverProperties {
+    private final static BrowserConfig config = ConfigFactory.create(BrowserConfig.class, System.getProperties());
+    private final static String CHROME_DRIVER_PATH = "src\\main\\resources\\driver\\chromedriver.exe";
+    private final static String EDGE_DRIVER_PATH = "src\\main\\resources\\driver\\msedgedriver.exe";
```

Comment:
it's much easier with https://bonigarcia.dev/webdrivermanager/

Reply:
rewrite with wdm